Makefile.am

```diff
@@ -24,4 +24,22 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR
 # IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-SUBDIRS = src tests prof 
+SUBDIRS = src tests prof
+
+# Debian packaging
+dpkg_dir = packaging
+dist_src = $(PACKAGE)-$(VERSION).tar.gz
+deb_src  = $(PACKAGE)_$(VERSION).orig.tar.gz
+
+deb-package:
+	@which "$(DEBUILD)" 2>&1 > /dev/null || { echo "No debuild executable found" 1>&2 && exit 1; }
+	rm -rf $(dpkg_dir)
+	$(MKDIR_P) $(dpkg_dir)
+	$(MAKE) dist
+	tar -xvf $(dist_src) -C $(dpkg_dir)
```

Comment:
Generally yes though GNU tar (and it is more or less guaranteed to be gnu tar since we are building deb) is smart enough to figure that out based on the extension. Typically I would just throw -xvf.  Old BSD tar is a different story of course. In any case I added 'z', it does not hurt.